reo/nested_inputs.py

```diff
@@ -286,7 +286,7 @@ def list_of_dict(input):
         "type": "float",
         "min": 0.0,
         "max": 1.0e6,
-        "description": "Land area in acres available for PV panel siting"
+        "description": "Land area in acres available for PV panel and wind turbine siting"
```

Comment:
wind is not currently being constrained by land_acres

Reply:
@hdunham I believe it is in v2, [here](https://github.com/NREL/REopt_API/blob/4b1a154ab82380f39ee16d12b6047e279f67a944/reo/src/data_manager.py#L1036). 

It is _not_ in v3/Julia Pkg, currently, but we've added an issue for this [here](https://github.com/NREL/REopt.jl/issues/108)

Reply:
So then can we not edit this description until we actually make the change?

Reply:
@hdunham this change is in `reo/nested_inputs.py`, i.e. `v2`, and it is accurately describing v2 behavior. Am I missing something?

Reply:
Nope it was me that was missing that this change was in reo/